recipes/mysql-connector-cpp/all/conanfile.py

```diff
@@ -0,0 +1,49 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake
+
+
+class MysqlConnectorCppConan(ConanFile):
+    name = "mysql-connector-cpp"
+    version = "8.0.27"
+    license = "GPLv2"
+    author = "Oracle"
+    url = "https://dev.mysql.com/doc/connector-cpp/en/"
```

Comment:
for `url` in we use the URL of _this_ repository (`conan-center-index` in GitHub), while the `homepage` attribute points to that of the library itself